rfcs/0005-pablo-distribution.md

```diff
@@ -0,0 +1,667 @@
+
+To compensate for this new adjustment, a reduction <img src="0005-pablo-distribution-assets/images/stem-7c4ec4f9c189cb8f3edb39740e43c33f.png" width="16" height="10" alt="stem 7c4ec4f9c189cb8f3edb39740e43c33f" /></span>
+of reward for each staker needs to be tracked,
+
+<img src="0005-pablo-distribution-assets/images/stem-828ec270409cb6ff5cfc583587d0eae9.png" width="142" height="18" alt="stem 828ec270409cb6ff5cfc583587d0eae9" /></span>
+← (1)
+
+In general,
+
+<img src="0005-pablo-distribution-assets/images/stem-0f2f030a4f8a3c172e968af2768a3ec8.png" width="349" height="11" alt="stem 0f2f030a4f8a3c172e968af2768a3ec8" /></span>
+
+##### 5.3.1.2. When removing a staker from the pool the above addition step has to be reverted
+
+This can be done by subtracting the <img src="0005-pablo-distribution-assets/images/stem-7c4ec4f9c189cb8f3edb39740e43c33f.png" width="16" height="10" alt="stem 7c4ec4f9c189cb8f3edb39740e43c33f" /></span>
+of the staker from the total pool and reducing the pool shares
+accordingly.
+
+##### 5.3.1.3. When adding a new reward to the pool the calculations remain the same other than increasing the reward pool as follows,
+
+<img src="0005-pablo-distribution-assets/images/stem-26132ac9393fe54200c2208dc9244ea4.png" width="160" height="11" alt="stem 26132ac9393fe54200c2208dc9244ea4" /></span>
+
+Since already claimed rewards(<img src="0005-pablo-distribution-assets/images/stem-7c4ec4f9c189cb8f3edb39740e43c33f.png" width="16" height="10" alt="stem 7c4ec4f9c189cb8f3edb39740e43c33f" /></span>)
+are tracked for each staker, they can always claim the new reward share
+from <img src="0005-pablo-distribution-assets/images/stem-32efe856de4078991a47242cc1d89349.png" width="43" height="11" alt="stem 32efe856de4078991a47242cc1d89349" /></span>
+later.
+
+##### 5.3.1.4. When extending an existing position
+
+Extension of an existing staker position can be treated in the same way
+as adding a new staker as the following relationship holds with the new
+stake <img src="0005-pablo-distribution-assets/images/stem-9849cee8ec3e29bf6d2ea80a64d995dd.png" width="23" height="10" alt="stem 9849cee8ec3e29bf6d2ea80a64d995dd" /></span>
+and the corresponding inflation <img src="0005-pablo-distribution-assets/images/stem-e7d319c4dcb739d8e91edd37454e20e8.png" width="25" height="10" alt="stem e7d319c4dcb739d8e91edd37454e20e8" /></span>,
+
+new staker to add <img src="0005-pablo-distribution-assets/images/stem-1b75a50a55357d9a7a8d3ecbb06df470.png" width="180" height="20" alt="stem 1b75a50a55357d9a7a8d3ecbb06df470" /></span>
+← (2)
+
+Now with (1) + (2),
+
+<img src="0005-pablo-distribution-assets/images/stem-6a000622e842e98de57502915826da7b.png" width="245" height="20" alt="stem 6a000622e842e98de57502915826da7b" /></span>
+
+Therefore, same computation as before with <img src="0005-pablo-distribution-assets/images/stem-9849cee8ec3e29bf6d2ea80a64d995dd.png" width="23" height="10" alt="stem 9849cee8ec3e29bf6d2ea80a64d995dd" /></span>
```

Comment:
no sure that i see this formula works well . imagine i added 100 PBLO for 4 years. than in 2 years he adds 1000000000 PBLO. but its leverage will be like for 4 years. really he locked for 2 years.

Reply:
I will clarify, we need to avoid boosting for extend position operations.